### PR TITLE
Added test and path for email banner

### PR DESCRIPTION
**Pull request by darrenb-ta**
- New test added to existing test.spec
- Added the "Please enter email address" banner Xpath to the login.screen so test can be verified, user cannot login with null vals in email / password fields

### CODE REVIEW

File: test/specs/eg-access/test.spec.js [hunk 1]
```diff
@@ -37,4 +37,8 @@ describe('Testing the eg access app', () => {
         Login.EGAccess();
         ActionHelper.isVisible(LoginForm.EventSearchInput);
     });
+    it('Verify the user cannot login with null values in username / password input boxes.', () => {
```
Comment (pirasanthan-jesugeevegan) [inline test/specs/eg-access/test.spec.js:40]: shouldn't it be `&` instead of `/`

**Review by pirasanthan-jesugeevegan — APPROVED** [review]

**Review by Pelumi-Festicket — APPROVED** [review]

**Comment by Pelumi-Festicket** [comment]
Great work Darren 👏🏾

**Review by Pelumi-Festicket — APPROVED** [review]
Great work Darren 👏🏾

**Review by Pelumi-Festicket — APPROVED** [review]

**Comment by swetha-golipally** [comment]
Reviewed and looks fine.

**Review by swetha-golipally — APPROVED** [review]

**Pull request merged**